docker-compose.uffizzi.yml

```diff
@@ -0,0 +1,19 @@
+version: '3'
+
+x-uffizzi:
+  ingress:
+    service: application
+    port: 6006
+
+services:
+  application:
+    image: "${APP_IMAGE}"
+    ports:
+      - "6006:6006"
+    restart: always
```

Comment:
I believe Uffizzi ignores this `restart` specification. But Kubernetes does always try to restart containers.

Reply:
OK, I will remove it.